### ISSUE TITLE
Crashes on too many plugin load requests

### ISSUE THREAD

**Issue by JohnnyErnest** [stevenknox/PluginsDemo#1]
<img width="305" alt="PluginDemoError" src="https://user-images.githubusercontent.com/25945156/80199301-c6a69680-85e6-11ea-9689-82ad11389659.png">

Happens on the third time or so that I run the Hello World demo or the time lookup. Seems to be occurring on RunGarbageCollection(); when I did some debugging.

I tried adding WeakReference to test per the MS Docs below, but still got the same error somewhere between 3-7 runs of clicking the Hello World demo button. 

https://docs.microsoft.com/en-us/dotnet/standard/assembly/unloadability

**Comment by HybridSolutions** [comment]
Yeah I confirme this as well. It only happens when debugging. Not when published.
